packages/webapp/src/components/Form/Button/index.tsx

```diff
@@ -1,9 +1,24 @@
+/*
+ *  Copyright 2022-2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
 import React, { ReactNode } from 'react';
 import styles from './button.module.scss';
 import clsx from 'clsx';
 
 type ButtonProps = {
-  color?: 'primary' | 'secondary' | 'success' | 'error' | 'warning' | 'none';
+  color?: 'primary' | 'secondary' | 'secondary-2' | 'secondary-cta' | 'none';
```

Comment:
You might have noticed, but `warning` is being used too.

![Screenshot 2024-03-27 at 3 35 26 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/71dc3e07-5be6-4e64-a699-107a33665611)

Reply:
I did indeed! 🙂